### DISCUSSION TITLE
Does anyone know what the immediate modify setting do?

### DISCUSSION THREAD

**Discussion by GitExploraa** [darkreader/darkreader#9084] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/44654056/173826046-959f7aed-85b1-46ab-878e-74701dd80868.png)

 Also it'd be about time to update  the Help page, since a lot of Settings descriptions are missing from there

**Comment by Gusted** [comment, marked as answer]
See, #8322 

> Also it'd be about time to update the Help page, since a lot of Settings descriptions are missing from there

The darkreader.org site currently only reflects the normal UI.

**Reply by GitExploraa** [reply]
Thanks for the answer, and hopefully the site does get updated to reflect the new changes down the line
